src/scripts/modules/inline-embed.js

```diff
@@ -1,15 +1,78 @@
-const init = () => {
-  Array.from(document.querySelectorAll('.inlineVideo'))
-    .forEach((video) => {
-      video.addEventListener('click', (e) => {
-        e.preventDefault();
+const isUrl = str => str.indexOf('http') === 0;
+const isEmbed = str => isUrl(str) && (str.indexOf('embed') !== -1 || str.indexOf('player') !== -1);
+const isID = str => !isEmbed(str);
+const isVimeo = str => (isUrl(str) && str.indexOf('vimeo') !== -1) || (isID(str) && /^\d+$/.test(str));
+const isYoutube = str => (isUrl(str) && str.indexOf('youtube') !== -1) || isID(str);
+const el = tagName => document.createElement(tagName);
+
+const getEmbedUrl = (embed, autoplay = false) => {
+  // https://player.vimeo.com/video/000000000/?api=1
+  // https://www.youtube.com/embed/kjahsdlkaj/?enablejsapi=1&html5=1
+  if (isEmbed(embed)) {
+    return `${embed}${autoplay ? '?autoplay=1' : ''}`;
```

Comment:
There are quite a few references to `${autoplay ? '?autoplay=1' : ''}`. It may be worth assigning the value at the top of the function, so that you're only performing the conditional operation a single time.